data_sync/README.md

```diff
@@ -1,5 +1,7 @@
 # Data Sync
 
+The data sync protocol defines the protocol with which clients interact. It is the highest protocol in the stack and communicates directly with the secure transport layer. 
```

Comment:
with which `data sync clients` interact.

Not highest protocol inthe stack, that would be data sync clients

I'd say it interacts directly with secure transport layer, not communicates (sounds off to me but maybe it's fine)

Reply:
Would you consider data sync clients a protocol, I would consider it the interaction application for big brother in general, outside of the actual family.